README.md

```diff
@@ -117,6 +117,17 @@ chatkit_integration_locator=<INSTANCE_LOCATOR>
 chatkit_integration_key=<SECRET_KEY>
 ```
 
+## Linting
+
+We currently use [ktlint](https://github.com/pinterest/ktlint).
+You can use `ktlintCheck` to check if there are any errors.
```

Comment:
This is `./gradlew ktlintCheck`?

Reply:
so they're available as tasks inside AS, to run them in the command line you'd have to do `./gradlew <module_name>:ktlintCheck` e.g. `./gradlew chatkit-android:ktlintCheck`

Reply:
I can update the readme to explain that more clearly?